ddp-workspace/projects/ddp-dsm-ui/src/app/participant-list/participant-list.model.ts

```diff
@@ -39,7 +39,9 @@ export class Participant {
     if (jsonData != null) {
       jsonData.forEach((val) => {
         const oncHistory = OncHistoryDetail.parse(val);
-        oncHistoryDetails.push(oncHistory);
+        if (oncHistory) {
+          oncHistoryDetails.push(oncHistory);
```

Comment:
if onc history is deleted the parse will return null and so don't add that in the array and don't push it